src/server/cea_608_encoder/caption_string_utility.py

```diff
@@ -141,20 +144,44 @@ def create_byte_pairs_for_caption_string(caption_string: str) -> list:
     byte_pairs = bytes_to_byte_pairs(raw_hex_values)
     return byte_pairs
 
+def create_byte_pairs_for_midrow_style(color: str, underline = False):
+    byte_list = []
+    
+    # Default: text color: white, no underline
+    first_byte = 0x11 
+    second_byte = 0x20
+
+    if isinstance(color, str):
+        color = color.lower()
+    if color in text_colors:
+        second_byte += text_colors[color]
+    if underline == True: 
+        second_byte += 0x1  
+
+    bytes = []
```

Comment:
why do we need to allocate a new list here if we have bytes_list defined at the top of the function to use?

Reply:
Because it did not allow me to store integers and hex values in the same list. Thus, the function to check parity bit would give me errors. I've allocated the new list just to decrease the amount of lines of code, and avoid repeating this 4 times:
if check_parity(first_byte) == 0:
                first_byte = add_parity_to_byte(first_byte)
            byte_list.append(hex(first_byte))

Reply:
Removed the "bytes" list.